lib/pages/new_space/new_space.dart

```diff
@@ -150,26 +151,18 @@ class NewSpaceController extends State<NewSpace> {
     try {
       final avatar = this.avatar;
       avatarUrl ??= avatar == null ? null : await client.uploadContent(avatar);
-
+      final classCode = await SpaceCodeUtil.generateSpaceCode(client);
+      if (classCode == null) {
+        return;
+      }
       final spaceId = await client.createRoom(
         // #Pangea
-        // preset: publicGroup
```

Comment:
Can you replace this? It's original FluffyChat code that's commented out. Also, in files that are originally from fluffychat (anything that's not in the /pangea folder) can you wrap any changes / additions in ```// #Pangea``` ```// Pangea#``` comments?